.changeset/chilled-lemons-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@optiaxiom/react": minor
```

Comment:
should be a patch type change - since we're still on v0 - so while this is a new backwards compatible feature - for now we're considering both bug fixes and features as patch - and breaking changes as minor
```suggestion
"@optiaxiom/react": patch
```

Reply:
Got it vai.